### DISCUSSION TITLE
How to enable U2F on teleport cloud?

### DISCUSSION THREAD

**Discussion by emiel-phinion** [gravitational/teleport#7906] [Q&A] [answered]
I'm trying to enable U2F support in addition to TOTP according to the guide at https://goteleport.com/docs/access-controls/guides/u2f/

The guide tells me to edit the teleport.yaml file, but as far as I can tell there is no teleport.yaml file for the cluster that I can edit in teleport cloud. 

How can I enable U2F on teleport cloud?

**Comment by klizhentas** [comment, marked as answer]
Please take a look at  [this guide](https://goteleport.com/docs/cloud/guides/u2f/)

**Reply by emiel-phinion** [reply]
Thanks! That was the missing piece.
